### ISSUE TITLE
Enhance contrast: search field placeholder text and form input borders

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/lively#20]
Placeholder text for both full text search and text field elements on the advanced item search is below minimum contrast requirements, clocking in at 1.6:1 (#CCCCCC on #FFFFFF). The contrast must be increased to at least 4.5:1 for text at this size (#767676 works here to meet this requirement).

Text field and drop-down form input borders, which apply the same color, are also below contrast requirements. This should be increased to at least 3:1 for these elements (#949494 satisfies this).

**Comment by aviannamiller** [comment]
Changes look great, though both the `#advanced-search .field .numeric-toggle-time` and `button.add-value,button.remove-value` styles are not receiving the field-border color changes. For improved UI accessibility, it would be great to apply that updated border color to these as well.
